astropy/visualization/hist.py

```diff
@@ -31,11 +31,11 @@ def hist(x, bins=10, ax=None, max_bins=1e5, **kwargs):
 
         - 'freedman' : use the Freedman-Diaconis rule to determine bins
 
-    ax : Axes instance (optional)
+    ax : Axes instance, optional
```

Comment:
Do you not want API link to `Axes` here?

Reply:
Yes we probably should do this - I think there will be other cases of this though, so let's do that in a separate PR